src/js/mock_runtime.js

```diff
@@ -75,3 +75,16 @@ global.DoubleSubFails = () => {
 global.SnapshotBug = () => {
   assert("1,2,3" === String([1, 2, 3]));
 };
+
+global.ErrorHandling = () => {
+  global.onerror = (message, source, line, col, error) => {
+    assert("Error: xxx" === message);
+    assert(source.endsWith("js/mock_runtime.js"));
+    deno.print(`line ${line} col ${col}`);
+    assert(line === 89);
```

Comment:
Nice -- that's gonna be very brittle though.
Might be a little bit more future proof to do `eval("\n\n  notdefined()")` and assert that the error is at [2, 2].

Reply:
Good suggestion. Done.